### DISCUSSION TITLE
Custom Blockchain RPC endpoint

### DISCUSSION THREAD

**Discussion by cod3d** [WalletConnect/walletconnect-monorepo#3758] [Developer Support]
I have a concern regarding the RPC configuration with Web3Modal and WalletConnect. In my application, I provided a custom configuration for my RPC, as shown below: 

https://pastebin.com/ziUjqZbA (I cannot format code here)
    
However, even with this configuration, it seems that Web3Modal still defaults to using https://rpc.walletconnect.com/v1 for WalletConnect. There's no mention of this URL in my configuration. Why does Web3Modal continue to use this endpoint? How can I ensure my custom RPC is utilized? For example this https://github.com/WalletConnect/blockchain-api

**Comment by cod3d** [comment]
It seems I must rebuild the web3modal project with my own urls :0
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/48947806/a3140f77-59c8-4a68-9c60-466ccc90c19d)

**Reply by glitch-txs** [reply]
hi, could you show your configuration settings?
